_tuts/2-structured-data.diff

```diff
@@ -3184,76 +3183,6 @@ index 0000000..d2f9240
 +    "symfony/http-kernel": "^3.0"
 +  }
 +}
-diff --git a/test/e2e/BookshelfTest.php b/test/e2e/BookshelfTest.php
-deleted file mode 100644
-index 769e07d..0000000
---- a/test/e2e/BookshelfTest.php
-+++ /dev/null
-@@ -1,64 +0,0 @@
--<?php
--/*
-- * Copyright 2015 Google Inc. All Rights Reserved.
-- *
-- * Licensed under the Apache License, Version 2.0 (the "License");
-- * you may not use this file except in compliance with the License.
-- * You may obtain a copy of the License at
-- *
-- *   http://www.apache.org/licenses/LICENSE-2.0
-- *
-- * Unless required by applicable law or agreed to in writing, software
-- * distributed under the License is distributed on an "AS IS" BASIS,
-- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-- * See the License for the specific language governing permissions and
-- * limitations under the License.
-- */
--
--namespace Google\Cloud\Samples\Bookshelf;
--
--use Behat\Mink\Driver\GoutteDriver;
--use Behat\Mink\Session;
--
--/**
-- * Class BookshelfTest
-- */
--class BookshelfTest extends \PHPUnit_Framework_TestCase
```

Comment:
Are these changes expected? Essentially we are removing `BookshelfTest` and adding `E2eTest`...

I feel like this is a bug / error.